app/Models/Project.php

```diff
@@ -78,6 +78,7 @@ class Project extends Model
      * @var array
      */
     protected $casts = [
+        'name' => 'array',
```

Comment:
Is this an Array because of the JSON object stored in the DB for translations?

Reply:
Yep, see: https://laravel.com/docs/8.x/eloquent-mutators#array-and-json-casting